gov/staker/staker.gno

```diff
@@ -67,6 +67,7 @@ func Delegate(to std.Address, amount uint64) {
 func Redelegate(from, to std.Address, amount uint64) {
 	en.MintAndDistributeGns()
 	calculateReward()
+	CollectReward()
```

Comment:
When CollectReward function is called, calculateReward function is also called.
So, i suggest that calculateReward function should remove when CollectReward function is called to avoid duplicated function call

Reply:
b56b63a950d6c6d9059f582a2bb722305cdb57e2